get-pmm.sh

```diff
@@ -260,13 +266,14 @@ show_message() {
     die "${RED}ERROR: cannot detect PMM server address${NOFORMAT}"
```

Comment:
Do we need `\n` here?

Reply:
Yes, missed that one when I changed the default echo behavior.